docs/topics/db/sql.txt

```diff
@@ -23,9 +23,8 @@ __ `executing custom SQL directly`_
       :doc:`custom query expressions </ref/models/expressions/>`.
 
     Before using raw SQL, explore :doc:`the ORM </topics/db/index>`. Ask on
-    |django-users| or the `#django IRC channel
-    <https://webchat.freenode.net/#django>`_ to see if the ORM supports your
-    use case.
+    |django-users| or the `#django IRC channel <irc://irc.libera.chat/django>`_
+    to see if the ORM supports your use case.
```

Comment:
Whilst we're at it, this would probably be better directing users to the "FAQ: Getting Help" page so they also see the forum as a channel, rather than duplicating the info.

Reply:
Updated.